slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c

```diff
@@ -23,6 +23,7 @@
 
 // Global variable from slsDetectorServer_funcs
 extern int debugflag;
+extern int updateFlag;
```

Comment:
I don't have a better name but could we name this so it tells that we are skipping the setup?